### DISCUSSION TITLE
Requirements to a server for KnowWE

### DISCUSSION THREAD

**Discussion by VitaliyMez** [denkbares/d3web-KnowWE#11] [Q&A] [answered]
Hi, can you please share the min/sufficient requirements for the VPS (ram/cpu/storage) to deploy KnowWE.
Thank you!

**Comment by AlbrechtStriffler** [comment, marked as answer]
Basically any hardware that can run Java and Tomcat will do. The requirements heavily depend on the size of the wiki to be developed. For example, the wiki provided in the distribution should run with any CPU and 1GB RAM, few hundred MB disk space. But we also have wikis that require 10GB of RAM and fast multi-core CPUs.

If you are setting up a new server, I would recommand something like a 2 or 4 core CPU, 2GB available RAM and a few GB disk space. With the option to add more.
